### PR TITLE
Swift package manager support

### PR DESCRIPTION
**Pull request by adriana-elizondo**
## What it does

**Adds swift package manager support to Kanvas:**
- Created swift package
- Added required imports

## How to test

- Open package file and build.

### CODE REVIEW

**Comment by gmazzei** [comment]
@adriana-elizondo Hey 👋  Sorry for the delay! 
I'll add @bjtitus as the reviewer, who has been checking most PRs recently.

File: Sources/kanvas-ios/kanvas_ios.swift [hunk 1]
```diff
@@ -0,0 +1,3 @@
+struct kanvas_ios {
+    var text = "Hello, World!"
```
Comment (bjtitus) [inline Sources/kanvas-ios/kanvas_ios.swift:2]: Do we need this file?

File: Tests/kanvas-iosTests/XCTestManifests.swift [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import XCTest
```
Comment (bjtitus) [inline Tests/kanvas-iosTests/XCTestManifests.swift:1]: Should we include this if we have no tests?

**Comment by bjtitus** [comment]
This seemed to work well for me with a few testing methods:

* Built manually using: `bash -c swift test -Xswiftc "-sdk" -Xswiftc "`xcrun --sdk iphonesimulator --show-sdk-path`" -Xswiftc "-target" -Xswiftc "x86_64-apple-ios14.3-simulator"` (I use fish, which is why I used the bash command).
* Added the Swift package by dragging and dropping the folder into a new project and adding as a library dependency.

I left a few minor thoughts but the main thing I wonder is whether we should add something building or verifying the package to our CI process to ensure that future files don't break the package.

**Pull request closed without merging**